src/utils/video-img_2.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+import shutil
+from pathlib import Path
+
+# global variable
+VIDEO_PATH = Path("D:/GDP/TESTING/nofi001.mp4")
+EXTRACT_FOLDER = Path("D:/GDP/TESTING/vi-im/")
+
+def extract_frames(video_path, result_path, index, desired_fps):
+    '''
+        :param video_path: video path
+        :param result_path: the location where the frame image is stored
+        :param index: the starting sequence number of the frame save filename,
+                      this sequence number will be appended to the filename of each saved frame image
+        :param desired_fps
+        '''
+
+    video = cv2.VideoCapture() # read frames from video source
+    cap = cv2.VideoCapture(str(video_path))
+    fps = cap.get(cv2.CAP_PROP_FPS) # get the frame rate of the video
+    EXTRACT_FREQUENCY = int(fps / desired_fps) # Frame extraction frequency
+    print("EXTRACT_FREQUENCY: ", EXTRACT_FREQUENCY)
```

Comment:
we do not need verbose. It is good for debugging but will flood the actual console when we will be dealing with a lot of videos